telcell/pipeline.py

```diff
@@ -10,51 +10,34 @@
 
 def run_pipeline(
         data: Iterable[Tuple[Track, Track, Mapping[str, Any]]],
-        models: Iterable[Model],
+        model: Model,
         output_dir: Union[str, Path] = None,
-) -> Sequence[Sequence[float]]:
+) -> Sequence[float]:
     # If no `output_dir` was specified, use the current working directory.
     output_dir = Path(output_dir or ".")
 
-    # Keep track of all likelihood ratios computed by each model.
-    all_lrs = []
-
-    # TODO: discussion about the following:
```

Comment:
Using lrbenchmark forces this function to reuse the data. Might not be ideal, but it does make things simpler right now.